docs/dev/table/tableApi.zh.md

```diff
@@ -477,11 +477,11 @@ val result = orders.where($"b" === "red")
   <tbody>
   	<tr>
   		<td>
-        <strong>From</strong><br>
-        <span class="label label-primary">批处理</span> <span class="label label-primary">流处理</span>
+        <strong>FromPath</strong><br>
+        <span class="label label-primary">Batch</span> <span class="label label-primary">Streaming</span>
```

Comment:
why change this?

Reply:
@dianfu So sorry for this issue.
Thank you for your reminder.